### PR TITLE
Rename bounding box model to FieldBoundingBox

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/GetBoundingBoxes.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public static void main(String[] args) {
             final List<FormPage> pages = recognizedForm.getPages();
             for (int i1 = 0; i1 < pages.size(); i1++) {
                 final FormPage formPage = pages.get(i1);
-                System.out.printf("------- Recognizing info on page %s of Form -------%n", i1);
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/GetBoundingBoxes.java:59]: nit: not sure if we want to rename the sample file to reflect the change.

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java [hunk 2]
```diff
@@ -259,10 +259,10 @@ private static void validateFormWordData(List<TextWord> expectedFormWords,
         }
     }
 
-    private static void validateBoundingBoxData(List<Float> expectedBoundingBox, BoundingBox actualBoundingBox) {
-        if (actualBoundingBox != null && actualBoundingBox.getPoints() != null) {
+    private static void validateBoundingBoxData(List<Float> expectedBoundingBox, FieldBoundingBox actualFieldBoundingBox) {
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:262]: consider to update the method name and expected value name as well.

**Comment by maririos** [comment]
@samvaity are the build failures related to your change?

**Comment by samvaity** [comment]
>@samvaity are the build failures related to your change

Not related to the change but seeing some random failures. I had 1 green CI yesterday, just re-ran the failed tests

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormLine.java [hunk 3]
```diff
@@ -47,7 +47,7 @@ public FormLine(String text, BoundingBox boundingBox, Integer pageNumber,
      * {@inheritDoc}
      */
     @Override
-    public BoundingBox getBoundingBox() {
+    public FieldBoundingBox getBoundingBox() {
```
Comment (srnagar) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormLine.java:50]: `getFieldBoundingBox()`?

Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormLine.java:50]: We have actually decided to keep this as `getBoundingBox()`

**Pull request merged**